Palo Alto Networks/paloalto-cortex/ingest/parser.yml

```
@@ -0,0 +1,37 @@
+name: cortex-xdr
+pipeline:
+  - name: parsed_event
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: setter
+    description: ''
+    filter: ''
+    external: null
+stages:
+  setter:
+    actions:
+      - set:
+          cortexxdr.shost: '{{parsed_event.message.shost}}'
+          cortexxdr.suser: '{{parsed_event.message.suser}}'
+          cortexxdr.action: '{{parsed_event.message.act|lower}}'
+          cortexxdr.category: '{{parsed_event.message.cat|upper}}'
+          cortexxdr.event_subtype: '{{parsed_event.message.subtype}}'
+          cortexxdr.severity_human: '{{parsed_event.message.Severity}}'
+          cortexxdr.original_message: '{{parsed_event.message.msg}}'
+        filter: ''
+        name: set
+      - fallback: UNPARSED
+        dictionary:
+          '0': UNKNOWN
+          '6': LOW
+          '8': MEDIUM
+          '9': HIGH
+        mapping:
+          cortexxdr.severity_human: cortexxdr.severity_human
```

According to the suggestion above:
```suggestion
          event.severity: cortexxdr.severity_human
```